pkg/zson/docs/spec.md

```diff
@@ -358,20 +353,20 @@ record(
 )
 ```
 The special characters that must be escaped if they appear within a value are:
-* newline (\n)
-* backslash (\\)
-* the separator character (usually \t)
-* the set_separator character inside of set and vector values (usually ,)
-* unset_field (usually -) if it appears as a value not be interpreted as "unset",
-
-Similarly, a set with no values must be specified  by the empty_field string (usually "(empty)")
-to distinguish it from a set with a single value "" and this must be escaped if it
-is a single-element set with the value "(empty)", i.e., escaped as "\(empty)".
-
-When processing legacy values, a column of type string named "_path" is
-inserted into value provided a #path directive previously appeared in the
-stream.  The contents of this _path field is set to the string value indicted
-in the #path directive and all the other columns are shifted one space to
+* newline (`\n`)
+* backslash (`\`)
+* the `#separator` character (usually `\t`)
+* the `#set_separator` character inside of set and vector values (usually `,`)
+* `#unset_field` (usually `-`) if it appears as a value not be interpreted as "unset",
+
+Similarly, a `set` with no values must be specified by the `#empty_field` string (usually `(empty)`)
+to distinguish it from a `set` with a single value that's a zero-length string, and this must be escaped if it
+is a single-element set with the value `(empty)`, i.e., escaped as `\x28empty)`.
+
+When processing legacy values, a column of type `string` named `_path` is
+inserted into each value, provided a `#path` directive previously appeared in the
+stream.  The contents of this `_path` field is set to the string value indicated
+in the `#path` directive. It becomes the leftmost column in the value and all the other columns are shifted one space to
```

Comment:
We said previously how "Unlike json, the ordering of the columns is significant..." so I was explicit here about where the exploded `_path` is expected to appear.

Reply:
I think the statement you quoted refers to "regular values" but this bit applies to legacy values.  The question of whether column ordering is preserved by any processing that occurs after file parsing is outside the immediate scope of specifying the file format, though it could fit here as part of broader "zson semantics".

Reply:
Leaving this one as-is.